entities.json

```diff
@@ -1,4 +1,11 @@
 {
+    "alias_type": {
+        "cache": {
+            "id": 39,
```

Comment:
This id is the first argument to `pg_advisory_xact_lock`/`pg_try_advisory_xact_lock`, and is unique for every cache prefix. We could've probably generated these somehow from the prefixes, but I preferred being explicit.